### PR TITLE
blog: Add post on detachable tabs WIP

### PR DESCRIPTION
**Pull request by ricekot**
<details><summary>Screenshot (click to expand) </summary>

image

</details>

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: site/content/blog/2024-09-18-detachable-tabs/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+ZAP tabs can now be opened in new windows.
+This includes the tabs in the workspace, tree, and information panels.
+
+To pop out a tab into a new window, simply right click on it, and select "Move Tab to New Window".
+
+We hope that this addition to ZAP improves its usability and allows you to better manage your workflow.
+This should be especially useful for people with multiple monitors.
+
+Try out this change by [downloading the latest ZAP weekly](https://www.zaproxy.org/download/#weekly) and let us know what you think on the [ZAP User Group](https://groups.google.com/group/zaproxy-users)!
```
Comment (thc202) [inline site/content/blog/2024-09-18-detachable-tabs/index.md:27]: `/download/#weekly`

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
Will explain why WIP on Slack in a bit 😁

File: site/content/blog/2024-09-18-detachable-tabs/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "ZAP now has Detachable Tabs!"
```
Comment (psiinon) [inline site/content/blog/2024-09-18-detachable-tabs/index.md:2]: Ah, but it doesnt 😁 
ZAP as far as 99% of our users are concerned is the latest released version, which does not support detachable tabs.
If we're going to publish this blog before 2.16 then we have to make it very clear that its only dev/nightly/weekly that support this

File: site/content/blog/2024-09-18-detachable-tabs/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+- blog
+- update
+- ui
+date: "2024-09-18"
+authors:
+- akshath
+---
+
+![An image containing screenshots of two ZAP windows. One is the ZAP workbench with a highlighted context menu item on the Request-Response tab with the text "Move Tab to New Window". The other window is the Request-Response panel."](images/zap-detachable-tabs.png)
```
Comment (psiinon) [inline site/content/blog/2024-09-18-detachable-tabs/index.md:17]: Again, the text is misleading.

File: site/content/blog/2024-09-18-detachable-tabs/index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+ZAP tabs can now be opened in new windows.
+This includes the tabs in the workspace, tree, and information panels.
+
+To pop out a tab into a new window, simply right click on it, and select "Move Tab to New Window".
```
Comment (psiinon) [inline site/content/blog/2024-09-18-detachable-tabs/index.md:22]: How can they pop it back in? (I know the answer, but other people wont).
Does this work in all of the ZAP screen modes?
Are there any gotchas?

**Review by psiinon — CHANGES_REQUESTED** [review]

**Comment by psiinon** [comment]
Yeah, that would be fine.
I still think a post to the user group might be better at this state...

**Pull request closed without merging**